libs/application-generic/src/logging/index.ts

```diff
@@ -10,135 +11,85 @@ export function getErrorInterceptor(): NestInterceptor {
 }
 export { Logger, LoggerModule, PinoLogger, storage, Store, getLoggerToken };
 
-const loggingLevelArr = ['error', 'warn', 'info', 'verbose', 'debug'];
-
 const loggingLevelSet = {
-  error: 50,
-  warn: 40,
+  trace: 10,
+  debug: 20,
   info: 30,
-  verbose: 20,
-  debug: 10,
+  warn: 40,
+  error: 50,
+  fatal: 60,
+  none: 70,
```

Comment:
alignment with the pino logger levels with additional 'none' in case we want to silence the logs.